server/parsec/asgi/administration.py

```diff
@@ -755,3 +760,36 @@ async def administration_organization_sequester_service_update_config(
         status_code=200,
         content={},
     )
+
+
+class AdministrationUsersOrganizations(BaseModel):
+    user_email: str
```

Comment:
In Parsec, user refers to a user certificate with a UserID

Here we want to know all the organizations that contains a given email, so we should uses human (as in Parsec human correspond to the email+label couple)



so basically you should replace every occurence of user in this PR by human ^^